lua/modules/linear_bolzmann_solvers/lbs_solver/lbs_io_flux_moments.cc

```diff
@@ -94,7 +94,6 @@ LBSReadSourceMoments(lua_State* L)
   const auto file_base = LuaArg<std::string>(L, 2);
   auto single_file_flag = LuaArgOptional<bool>(L, 3, false);
 
-  // Get pointer to solver
```

Comment:
just curious why these comments have been removed

Reply:
I'm not sure. Those were removed by @zhardy-lanl in his fork that I got the original implementation from

Reply:
I removed the comments because it felt redundant to me to say we are getting a pointer to the solver when we are calling a routine called `GetStackItem` and assigning it to something that says `solver`. Feel free to add it back.

Reply:
Just leave it then unless anyone else chimes in with an objection

Reply:
"just leave it" as meaning "put the comment back" or "keep it as is, i.e., with the comment currently removed"

Reply:
Just leave it meaning leave the comment out and don't change the state of the branch to put it back in.